worldedit-bukkit/src/main/java/com/sk89q/worldedit/bukkit/BukkitBlockCommandSender.java

```diff
@@ -53,7 +53,7 @@ public BukkitBlockCommandSender(WorldEditPlugin plugin, BlockCommandSender sende
         this.plugin = plugin;
         this.sender = sender;
         this.location = BukkitAdapter.adapt(sender.getBlock().getLocation());
-        this.uuid = new UUID(location.toVector().toBlockPoint().hashCode(), location.getExtent().hashCode());
+        this.uuid = UUID.nameUUIDFromBytes(sender.getName().getBytes());
```

Comment:
We should use the Charset-specifying version of `getBytes()` so that session persistence is OS-independent.

Reply:
random thought: is this the same method MC uses to gen offline uuids? if so, is it possible on offline servers to make a commandblock named `OfflinePlayer:xyz` and have them share a session with a player named `xyz`? if so, do we care?

Reply:
I've resolved both of these issues